pkg/endpoints/pipeline.go

```diff
@@ -126,27 +126,43 @@ func (r Resource) getPipelineImpl(name, namespace string) (v1alpha1.Pipeline, er
 	return *pipeline, nil
 }
 
-/* Get all pipeline runs in a given namespace */
+/* Get all pipeline runs in a given namespace, filters based on query parameters */
 func (r Resource) getAllPipelineRuns(request *restful.Request, response *restful.Response) {
+	// Request Params
 	namespace := request.PathParameter("namespace")
-	pipelineruns := r.PipelineClient.TektonV1alpha1().PipelineRuns(namespace)
+	// Query Params
 	repository := request.QueryParameter("repository")
+	name := request.QueryParameter("name")
+	logging.Log.Debugf("In getAllPipelineRuns: namespace: `%s`, repository query: `%s`, repository query: `%s`", namespace, repository, name)
```

Comment:
Repo query twice instead of name query (only on the header part and not the variable used)

Reply:
What is the name is specified and not the repository?

Reply:
We could build up a log string with fmt.Sprintf based on what's being provided and add a bunch of ifs, yeah having just the name is perfectly valid, so is just the repository, so is both